resources/views/airlines/show.blade.php

```diff
@@ -0,0 +1,146 @@
+<x-layout>
+    <div class="px-4 sm:px-6 lg:px-8">
+        <div class="sm:flex sm:items-center">
+            <div class="sm:flex-auto">
+                <h1 class="text-base font-semibold leading-6 text-gray-900 text-xl">Airlines</h1>
+            </div>
+            <div class="mt-4 sm:ml-16 sm:mt-0 sm:flex-none">
+                <button id="open-create-modal-button" type="button" class="block rounded-md bg-indigo-600 px-3 py-2 text-center text-sm font-semibold text-white shadow-sm hover:bg-indigo-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600">Add airline</button>
+            </div>
+        </div>
+        <div class="mt-8 flow-root px-8">
+            <div class="-mx-4 -my-2 overflow-x-auto sm:-mx-6 lg:-mx-8">
+                <div class="inline-block min-w-full py-2 align-middle sm:px-6 lg:px-8">
+
+                    <table class="min-w-full divide-y divide-gray-300">
+                        <thead>
+                            <tr>
+                                <th scope="col" class="py-3.5 pl-4 pr-3 text-left text-sm font-semibold text-gray-900 sm:pl-0">Id</th>
```

Comment:
when using tailwind, we need to think of components not just to reuse logic, but to reuse css as well. If we decide to have those `th` with the centered texts, we would have to change aaaaalll of them, but if we were using a `<x-table-head />` blade component, it would only require to change one thing.
Consider creating components for these things, and other repeated elements like the add button